docs/components/modeler/desktop-modeler/process-applications.md

```diff
@@ -44,25 +50,31 @@ consumer-loan-application/
 └── README.md
 ```
 
-When you open a file in the modeler, the system implicitly determines whether it belongs to a process application by checking for the presence of a `.process-application` file in the same folder or a parent folder. If such a file is found, the process application is implicitly opened, indicated by the blue item in the status bar. All related files within the process application are made accessible for navigation. Similarly, when you close the file, the process application is implicitly closed.
+## Editor support for process applications
+
+When you open a file in Modeler, the system implicitly determines whether it belongs to a process application. It does so by checking for the presence of a `.process-application` file in the same folder or a parent folder. Within a process application, Modeler offers advanced navigation and intelligence.
```

Comment:
[☁️] Thinking out loud here. Do we have documented anywhere what `intelligence` means? I didn't find any docs around it. Might be worth linking this to a section that list some of the capabilities the editor provides in this aspect.

Reply:
Intelligence for example means assistance during linking, mentioned below, but will also include advanced data handling and cross-file validation in the future. Today, we offer _basic_ intelligence. But the fact that we have a search boundary (the process application root) allows us to expand and add more, in the future.

Reply:
```suggestion
When you open a file in Modeler, the system implicitly determines whether it belongs to a process application. It does so by checking for the presence of a `.process-application` file in the same folder or a parent folder. Within a process application, Modeler offers improved navigation and assistance.
```